apps/server/lib/bootstrap.ts

```diff
@@ -14,22 +14,14 @@ import type { core as coreType, JSONSchema } from '@balena/jellyfish-types';
 import { TriggeredActionContract } from '@balena/jellyfish-types/build/worker';
 import {
 	Contract,
-	SessionContract,
-	SessionData,
 	StreamChange,
 	TypeContract,
 } from '@balena/jellyfish-types/build/core';
 import { Transformer } from '@balena/jellyfish-worker/build/transformers';
+import { getActorKey } from '@balena/jellyfish-worker/build/utils';
```

Comment:
These sorts of deep imports are dangerous, because they are not part of the top-level interface the worker module provides. This means that the internal implementation of the worker module is now tightly coupled to this code.
What can we do to eliminate the need for this at all?
FYI we discussed this "actor key" stuff a few months ago on an arch call and came to the conclusion that it's not needed (I jotted down some details [here](https://jel.ly.fish/improvement-jip-stop-generating-sessions-for-users-in-worker-code-adf47d50-62b7-43ef-91ef-349529d597a4) ).